omnigibson/termination_conditions/termination_condition_base.py

```diff
@@ -91,6 +91,15 @@ def success(self):
         assert self._done is not None, "At least one step() must occur before success can be calculated!"
         return self._done and self._terminate_is_success
 
+    @property
+    def partial_success(self):
+        """
+        Returns:
+            float: partial success if supposed, -1.0 otherwise 
+        """
+        assert self._done is not None, "At least one step() must occur before partial_success can be calculated!"
+        return -1.0 
```

Comment:
This magic value of -1 is not good. Many metrics can reasonably return -1. We can return None. Why is this not supported here anyway?